charts/consul/templates/dns-proxy-clusterrole.yaml

```
@@ -0,0 +1,34 @@
+{{- if (or (and (ne (.Values.dns.proxy.enabled | toString) "-") .Values.dns.proxy.enabled) (and (eq (.Values.dns.proxy.enabled | toString) "-") .Values.global.enabled)) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "consul.fullname" . }}-dns-proxy
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: dns-proxy
+{{- if or .Values.global.acls.manageSystemACLs .Values.global.enablePodSecurityPolicies }}
```

PodSecurityPolicy is deprecated in kubernetes and I think removed at this point, so we can remove it from here
```suggestion
{{- if .Values.global.acls.manageSystemACLs }}
```